src/main/java/com/powsybl/openloadflow/network/LfNetwork.java

```diff
@@ -637,6 +637,26 @@ private void reportSize(ReportNode reportNode) {
             this, busesById.values().size(), branches.size());
     }
 
+    /**
+     * Reports the fictitious injection total if not null
+     */
+    public void reportFictitiousInjectionTotal(ReportNode reportNode) {
+        double fictitiousTargetPNormInf = getBuses().stream().mapToDouble(LfBus::getFictitiousInjectionTargetP)
+                .map(Math::abs)
+                .sum();
+        double fictitiousTargetQNormInf = getBuses().stream().mapToDouble(LfBus::getFictitiousInjectionTargetQ)
+                .map(Math::abs)
+                .sum();
+        long busCount = getBuses().stream().filter(b -> b.getFictitiousInjectionTargetP() != 0 || b.getFictitiousInjectionTargetQ() != 0).count();
+        if (fictitiousTargetPNormInf + fictitiousTargetQNormInf > 0) {
```

Comment:
- perf: do single loop over buses instead of three
- functional: not sure about Math::abs ... I would prefer having sum of signed. In the if at the end, just change to busCount > 0